tasks/src/image-to-image/data.ts

```diff
@@ -0,0 +1,60 @@
+import { type TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
```

Comment:
I think it makes sense to have only VIDIT and CelebA datasets, we look for datasets with very general use cases 🙂